library/Fission/AWS/Route53.hs

```diff
@@ -20,9 +20,9 @@ import           Network.IPFS.Gateway.Types
 import           Network.IPFS.CID.Types
 
 registerDomain ::
-  ( MonadRIO           cfg m
+  ( MonadReader        cfg m
```

Comment:
What happened to `MonadRIO`? Well, we're moving to more generality, so there's far fewer instances when we need both of them together.

Reply:
At one point, I thought `MonadRIO` was the single most important type in our codebase. Its gifts were boundless. You won't be forgotten _sheds single tear_